tests/conda_env/test_cli.py

```diff
@@ -77,26 +75,23 @@ def run_env_command(command, prefix, *arguments):
         prefix: The prefix, for remove and create
         *arguments: The extra arguments
     """
-    p = create_parser()
-    prefix = escape_for_winpath(prefix)
 
-    if arguments:
-        arguments = list(map(escape_for_winpath, arguments))
+    arguments = massage_arguments(arguments)
+    arguments.insert(0, command)
 
     if command is Commands.ENV_EXPORT:
-        command_line = "{0} -n {1} {2}".format(command, prefix, " ".join(arguments))
+        arguments[1:1] = ['-n', prefix]
     elif command is Commands.ENV_CREATE: # CREATE
-        if prefix :
-            command_line = "{0} -f {1} {2}".format(command, prefix, " ".join(arguments))
-        else:
-            command_line = "{0} {1}".format(command, " ".join(arguments))
+        if prefix:
+            arguments[1:1] = ['-f', prefix]
     elif command is Commands.ENV_REMOVE:  # REMOVE
-        command_line = "{0} --yes -n {1} {2}".format(command, prefix, " ".join(arguments))
+        arguments[1:1] = ['--yes', '-n', prefix]
```

Comment:
Is this a way to insert?  If so, could you please change to the .insert syntax?  This syntax is confusing to me.